requirements-dev.txt

```diff
@@ -1,8 +1,10 @@
 -e .
 bumpversion==0.5.3
-flake8==3.5.0
-isort==4.3.4
+flake8==3.7.8
+isort==4.3.21
 mkdocs==1.0.4
-pytest-cov==2.6.0
-pytest==3.8.2
-mock==2.0.0
+mock==3.0.5
+pytest-cov==2.8.1
+pytest==3.8.2; python_version == '2.7'
+pytest==4.6.6; python_version == '3.4'
+pytest==5.2.1; python_version > '3.4'
```

Comment:
🏆  I need some like that on [simple-settings](https://github.com/drgarcia1986/simple-settings)